uktrade/data-hub-frontend
# Company overview

## Description of change

Overview tab and page has been added to companies local nav. The overview tab contains a single card that give an overview of the business details.
![Screenshot 2023-03-02 at 14 16 15](https://user-images.githubusercontent.com/72826129/222453464-a9122cf5-1e90-4f93-a5c8-ecbd493d1194.png)

## Test instructions

You should see an overview tab which contains a card showing business details. Where the details do not exists these should display as Not set

## Screenshots

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
